common/components/internal-header/_internal-header.scss

```
@@ -1,311 +1,182 @@
+  margin-right: govuk-spacing(2);
+  position: relative;
+  top: -2px;
+  fill: #fff;
 }
 
-.app-header__logotype-crest-fallback-image {
-  width: 42px;
-  height: 35px;
-  border: 0;
-  vertical-align: middle;
+.app-header__logo {
+  padding-right: govuk-spacing(3);
+  display: flex;
+  align-items: center;
 }
 
-.app-header__product-name {
-  @include govuk-font($size: 24);
-  display: inline-table;
+.app-header__logotype {
+  margin-right: govuk-spacing(2);
+  @include govuk-font($size: 24, $weight: bold);
+  display: flex;
+  align-items: center;
 }
 
 .app-header__link {
-  text-decoration: none;
+  @include govuk-font($size: 19);
 
   &:link,
-  &:visited {
-    color: $govuk-header-link;
+  &:visited,
+  &:active {
+    color: #fff;
+    text-decoration: none;
   }
 
   &:hover {
     text-decoration: underline;
   }
 
-  // When focussed, the text colour needs to be darker to ensure that colour
-  // contrast is still acceptable
   &:focus {
-    @include govuk-focused-text;
-  }
-
-  // alphagov/govuk_template includes a specific a:link:focus selector
-  // designed to make unvisited links a slightly darker blue when focussed, so
-  // we need to override the text colour for that combination of selectors.
-  @include govuk-compatibility(govuk_template) {
-    &:link:focus {
-      @include govuk-text-colour;
-    }
-  }
-}
-
-.app-header__link--homepage {
-  // Font size needs to be set on the link so that the box sizing is correct
-  // in Firefox
-  @include govuk-font($size: false, $weight: bold);
-
-  display: inline;
-  font-size: 30px; // We don't have a mixin that produces 30px font size
-  line-height: 1;
-
-  @include mq ($from: desktop) {
-    display: inline-block;
-  }
-
-  &:link,
-  &:visited {
+    outline: 3px solid transparent;
+    color: $govuk-header-background;
+    background-color: #fd0;
+    box-shadow: 0 -2px #fd0, 0 4px $govuk-header-background;
     text-decoration: none;
-  }
 
-  &:hover,
-  &:active {
-    // Negate the added border
-    margin-bottom: -1px;
-    // Omitting colour will use default value of currentColor – if we
-    // specified currentColor explicitly IE8 would ignore this rule.
-    border-bottom: 1px solid;
-  }
-
-  // Remove any borders that show when focused and hovered.
-  &:focus {
-    margin-bottom: 0;
-    border-bottom: 0;
-  }
-}
-
-.app-header__link--service-name {
-  display: inline-block;
-  margin-bottom: govuk-spacing(2);
-  @include govuk-font($size: 24, $weight: bold);
-}
-
-.app-header__logo,
-.app-header__content {
-  box-sizing: border-box;
-}
-
-.app-header__logo {
-  padding-bottom: govuk-spacing(2);
-  padding-right: govuk-spacing(9);
-
-  @include mq ($from: desktop) {
-    width: 50%;
-    padding-right: $govuk-gutter-half;
-    float: left;
-    vertical-align: top;
-  }
-}
-
-.app-header__content {
-  @include mq ($from: desktop) {
-    width: 50%;
-    padding-left: $govuk-gutter-half;
-    float: left;
+    svg {
+      fill: $govuk-header-background;
```

Could we use this (from line 43):
```suggestion
      fill: currentColor;
```

This will mean that it will benefit from whatever the color defined is rather than having to be set separately.